### ISSUE TITLE
Problems tab at terminal, scans excluded folders

### ISSUE THREAD

**Issue by ngalfas** [openhab/openhab-vscode#200]
I noticed that the problems tab is reporting problems in a folder i have excluded from the explorer tab.   

## Expected Behavior
I have a rules folder for UNUSED/old rules that i don't need to show on the explorer tab.
I have excluded this folder through settings. It stops showing in the explorer tab.
I expect that the terminal-problems tab doesn't scan the excluded folder.

## Current Behavior
No matter what i try, the terminal - problems tab scans and reports all the contents of the excluded folder.

**Comment by Confectrician** [comment]
Hi,

Can you share how you have excluded this folder in the settings?
Just to get an impression on how we could catch that.

**Comment by ngalfas** [comment]
Yes.  In my workplace folder (OPENHAB_CONF) i have the folder "z-notused".

The workspace starts on samba share  X:\OPENHAB_CONF
![Image12](https://user-images.githubusercontent.com/22976096/77472986-6ec91400-6e1d-11ea-8270-21d6034d15aa.png)


On the .vscode folder of the workspace, i edited settings.json file
![Image11](https://user-images.githubusercontent.com/22976096/77472717-01b57e80-6e1d-11ea-8628-2d23be1b1f8e.png)


As a result the z-notused folder is not shown in the workspace tree structure.

Now, if i open the terminal, i see 194 problems reported. All related to z-notused folder files.
![Image13](https://user-images.githubusercontent.com/22976096/77473268-ded79a00-6e1d-11ea-993b-e915908ed8ce.png)

It is interesting that the error report the filepath of my FREEBSD iocage container.

**Comment by Confectrician** [comment]
>It is interesting that the error report the filepath of my FREEBSD iocage container.

I would guess you are using the remote lsp server.
We have many reports about this behavior and this will also be a problem for this issue.

Our vscode extension just "runs" the lsp server module and tells it to use your openHAB connection configuration.
Anything else happens in openHAB core, where the remote LSP server does the checks and reports back errors.
We have (by now) no way to take any influence on this and i am not sure how i could send the "ignored" folder from vscode to the openHAB instance currently.

**Comment by ngalfas** [comment]
Ok then, just brought it to your attention.

**Comment by Confectrician** [comment]
No need to close this one. :)
Its not that i don't want to solve it, its more that it isn't that easy and will not be possible on the short term. (Since i am the only regular contributor here currently and i am still no lsp expert.)

So we can keep the issue here for documentation.
